app/views/pages/140.html.erb

```diff
@@ -19,40 +19,40 @@
             <h3>
                 What are you interested in teaching?
             </h3>
-            <p>Computer Science</p>
-            <p><a href='/'>Change</a></p>
+            <p class="govuk-body">Computer Science</p>
+            <p class="govuk-body"><a href='/'>Change</a></p>
         </div>
 
         <div class="govuk-form-group">
             <h3>
                 When do you intend to start your teacher training?
             </h3>
-            <p>2020</p>
-            <p><a href='/'>Change</a></p>
+            <p class="govuk-body">2020</p>
+            <p class="govuk-body"><a href='/'>Change</a></p>
         </div>
 
         <div class="govuk-form-group">
             <h3>
                 What stage are you at with your degree?
             </h3>
-            <p>Undergraduate</p>
-            <p><a href='/'>Change</a></p>
+            <p class="govuk-body">Undergraduate</p>
+            <p class="govuk-body"><a href='/'>Change</a></p>
         </div>
 
         <div class="govuk-form-group">
             <h3>
                 What subject is your degree in?
             </h3>
-            <p>Archeology</p>
-            <p><a href='/'>Change</a></p>
+            <p class="govuk-body">Archeology</p>
```

Comment:
Might be worth sticking this in a partial/view component to DRY it up

Reply:
This page is just a prototype, It has to be redone in a completely different format, but good point.